app/src/main/java/net/bible/android/view/util/widget/WindowButtonWidget.kt

```diff
@@ -84,6 +84,8 @@ class WindowButtonWidget(
     }
 
     private fun updateBackground() {
+        val isPinnedWindow = if (window?.isPinMode == null) false else window?.isPinMode
```

Comment:
```suggestion
        val isPinnedWindow = window?.isPinMode == true
```

Reply:
I had trouble with something like this because where I was using it require a Boolean and I think isPinMode is a nullable Boolean. I dont think i tried this though. I will try it.